### ISSUE TITLE
Mobile regression: menu icon no longer sticky

### ISSUE THREAD

**Issue by zachallaun** [elixir-lang/ex_doc#1835]
On my phone so can't figure out where this was introduced, but I noticed that the menu icon no longer scrolls with the page. This is the case whether the menu is open or not (if you open the menu and scroll the page behind it, the icon scrolls out of view).

This is a pretty big UX regression on mobile, since you have to scroll all the way to the top to navigate elsewhere. 

(iOS 17.1.2)

**Comment by josevalim** [comment]
Good find. This is good to fix in terms of functionality, you just need to remove this rule at the bottom of layout.css:

```css
  body.sidebar-closed .sidebar-button {
    position: absolute;
  }
```

But we should probably add gradient + padding back to the top as well, so the button is not immediately overlapping the text. Or alternatively we always keep the hamburger+search visible.

**Comment by zachallaun** [comment]
> Or alternatively we always keep the hamburger+search visible.

I think this would be nice! There's a common UI pattern where a navbar hides when you scroll down and then appears when you scroll up slightly -- I think that would work nicely for this case (IIRC Livebook uses this?).

**Comment by josevalim** [comment]
Oh, I love this idea! PRs to add this will be very welcome. Then we show the whole search and everything!

**Comment by josevalim** [comment]
@zachallaun btw, if you don't plan to tackle this, it is not problem. I can ask Paulo Valim to do it once he is back from holidays.

**Comment by zachallaun** [comment]
@josevalim I don't know that I'll be able to tackle before Paulo would -- if he's able to when he's back, that would be great!

**Comment by josevalim** [comment]
This has been fixed in #1838. We shall have a new release sooner than later. :)
